frontend/src/App.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Stats from './components/stats';
+import Search from './components/search';
+
+class App extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      loading: true,
+      user: [],
+      name: null,
+      placeholder: null,
+    }
+  }
+
+  handleChange = (event) => {
+    this.setState({
+      placeholder: event.target.value
+    })
+  }
+
+  handleSubmit = (event) => {
+    this.setState({
+      name: this.state.placeholder
+    })
+    event.preventDefault()
+  }
+
+  componentDidMount(){
+    let userId = '/search/' + this.state.name;
+    fetch(userId)
+    .then(res => res.json())
+    .then((data) =>
+      this.setState({ user: data }))
+  }
```

Comment:
Now realizing that componentDidMount() can be deleted to remove the error message on the main page and the program would still function the same.

Reply:
You can still make changes after you've made a PR.

A PR is saying "I want to merge `this branch` into master"
If you add more commits to `this branch` then that will be reflected in the PR

Reply:
Ahh perfect I'm understanding how it works now. I will make some changes, starting with this one.